loopchain/peer/block_manager.py

```diff
@@ -53,20 +54,19 @@ class BlockManager:
     MAINNET = "cf43b3fd45981431a0e64f79d07bfcf703e064b73b802c5f32834eec72142190"
     TESTNET = "885b8021826f7e741be7f53bb95b48221e9ab263f377e997b2e47a7b8f4a2a8b"
 
-    def __init__(self, name: str, channel_manager, peer_id, channel_name, level_db_identity):
+    def __init__(self, name: str, channel_manager, peer_id, channel_name, store_identity):
         self.__channel_service: ChannelService = channel_manager
         self.__channel_name = channel_name
         self.__pre_validate_strategy = self.__pre_validate
         self.__peer_id = peer_id
-        self.__level_db = None
-        self.__level_db_path = ""
-        self.__level_db, self.__level_db_path = util.init_level_db(
-            level_db_identity=f"{level_db_identity}_{channel_name}",
-            allow_rename_path=False
-        )
+
+        key_value_store, key_value_store_path = util.init_default_key_value_store(f"{store_identity}_{channel_name}")
+        self.__key_value_store = key_value_store
+        self.__key_value_store_path = key_value_store_path
```

Comment:
Managing `key_value store` in two classes is not nessesary.
`get_key_value_store` has to return `key_value_store` from `blockchain` class.